tests/expected/uninit/fixme_unions.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Z uninit-checks
+
+//! Tests for handling potentially uninitialized memory access via unions.
+
+use std::ptr::addr_of;
+
+#[repr(C)]
+#[derive(Clone, Copy)]
+union U {
+    a: u16,
+    b: u32,
+}
+
+/// Reading padding data via simple union access if union is passed to another function.
+#[kani::proof]
+unsafe fn cross_function_union() {
```

Comment:
Can you please add `should_pass` / `should_fail` as suffix, or update the comment to state what the expectation is? Just to make it easier to debug later on. Thanks!